Sources/CubeFoundationSwiftUI/Text/FontWeight+Extensions.swift

```diff
@@ -62,4 +62,30 @@ extension Font.Weight: ExpressibleByIntegerLiteral {
             return "Unknown"
         }
     }
+
+    /// Map to `UIFont.Weight` equivalent
+    public var uiWeight: UIFont.Weight {
+        switch self {
+        case .black:
+            return .black
```

Comment:
Syntax Q, might be nicer 1 line each? But staying consistent with the file atm

Reply:
Yeah 1 line could be nice for this one! But I reckon it's nicer matching the rest of the file tbh. Happy either way.